test/libsolidity/smtCheckerTests/unsupported/assembly_1.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
+    function f(uint x, uint y) public pure {
+        assembly {}
+        assert(x < y);
+    }
+
+    function g(uint x, uint y) public pure {
+        assembly {}
+        assert(x < y);
+    }
+}
+// ====
+// SMTEngine: all
+// SMTShowUnsupported: no
```

Comment:
Is there a test case with `SMTShowUnsupported: yes`?

Reply:
Yes, all the others by default.